File: src/main/java/com/github/wolfgenerals/petphraseplus/config/Command.java
package com.github.wolfgenerals.petphraseplus.config;

import net.fabricmc.fabric.api.client.command.v1.ClientCommandManager;
import net.minecraft.client.MinecraftClient;

public class Command {
    private static boolean needOpen = false;

    public static void register() {
        ClientCommandManager.DISPATCHER.register(
                ClientCommandManager.literal("petphraseplus").executes(context -> {
                            needOpen = true;
                            return 1;
                        }
                )
        );
    }

    public static void openScreen(MinecraftClient client) {
        if (needOpen) {
            ConfigSL.loadConfig();
            MinecraftClient.getInstance().setScreen(ConfigScreen.configScreen(client.currentScreen));
            needOpen = false;
        }
    }
}

File: src/main/kotlin/com/github/wolfgenerals/petphraseplus/config/Command.kt
package com.github.wolfgenerals.petphraseplus.config

import net.fabricmc.fabric.api.client.command.v1.ClientCommandManager
import net.fabricmc.fabric.api.client.event.lifecycle.v1.ClientTickEvents

fun registerCommand() {
    var needOpen = false
    ClientCommandManager.DISPATCHER.register(
        ClientCommandManager
            .literal("petphraseplus")
            .executes {
                needOpen = true
                1
            }
    )
    ClientTickEvents.END_CLIENT_TICK.register {
        if (!needOpen) return@register
        openScreen(it)
        needOpen = false
    }
}

